api4/cloud.go

```diff
@@ -486,6 +487,22 @@ func handleCWSWebhook(c *Context, w http.ResponseWriter, r *http.Request) {
 			c.Err = appErr
 			return
 		}
+	case model.EventTypeSubscriptionChanged:
+		// event.ProductLimits is nil if there was no change
+		if event.ProductLimits != nil {
+			c.App.GetPluginsEnvironment()
+			if pluginsEnvironment := c.App.GetPluginsEnvironment(); pluginsEnvironment != nil {
+				pluginsEnvironment.RunMultiPluginHook(func(hooks plugin.Hooks) bool {
```

Comment:
Interesting... TIL about this plugin hooks 👍

Reply:
Me too! Except YIL (yesterday I learned)